packages/devtools-reps/src/object-inspector/components/ObjectInspectorItem.js

```diff
@@ -266,7 +266,14 @@ class ObjectInspectorItem extends Component<Props> {
       return null;
     }
 
-    const { item, depth, focused, expanded, onLabelClick } = this.props;
+    const {
+      item,
+      depth,
+      focused,
+      expanded,
+      onLabelClick,
+      onExpressionDblClick
```

Comment:
I think the naming here is not ideal, since the ObjectInspector is a generic component. Also, it's then bound to `onDoubleClick`, so it could be a bit confusing.
Finally, there's already a `onDoubleClick` props on the `ObjectInspectorItem`, can't we use it?

Reply:
Would it be better if I rename `onExpressionDblClick` to `onDoubleClick`?

Reply:
@nchevobbe Could you provide a preference here?  This one is almost good to go! 👍

Reply:
Thank you for your patience @lawrencek0 !
So, I think we should try to **not** modify this file at all, and use the `onDoubleClick` prop it already provides.

So, in `Expression.js`, we would do: 
```js
onDoubleClick={(items, options) => {
  const {depth} = options;
  if (depth > 0) {
    return;
  }

  this.editExpression(expression, index)
}}
```

We would have the same functionality, without much changes.
Let me know if you have any question, I'll try to be more responsive this time!

Reply:
Thanks, this looks really neat! I always wondered what the options were for.